File: python/python-read-all-numbers-from-file/example3.py
# https://codevscolor.com/python-read-all-numbers-from-file

try:
    sum = 0
    given_file = open("input.txt", "r")

    lines = given_file.readlines()

    for line in lines:
        for c in line:
            if c.isdigit() == True:
                sum += int(c)

    print(f"Sum: {sum}")
    
    given_file.close()
except FileNotFoundError as e:
    print(e)

File: cpp/c-plus-plus-print-odd-numbers/example3.cpp
// https://codevscolor.com/c-plus-plus-print-odd-numbers
#include <iostream>
using namespace std;

int main()
{
    int i = 1;

    do
    {
        if (i % 2 == 1)
        {
            cout << i << " ";
        }
        i++;
    } while (i <= 100);

    return 0;
}